policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit.py

```diff
@@ -47,4 +47,18 @@ def formula(tax_unit, period, parameters):
 
         cap = max_(head_cap, spouse_cap)
 
-        return min_(uncapped_credit, cap)
+        credit = min_(uncapped_credit, cap)
+
+        # compute permanently and totally disabled veterans's credit
+        disabled_veterans = add(
+            tax_unit, period, ["is_fully_disabled_service_connected_veteran"]
+        )
+        permanently_disabled_veterans = add(
+            tax_unit, period, ["is_permanently_disabled_veteran"]
+        )
+        qualified_veterans = (
+            disabled_veterans and permanently_disabled_veterans
+        )
```

Comment:
```suggestion
        person = tax_unit.members
        disabled_service_connected_veteran = person("is_fully_disabled_service_connected_veteran", period)
        permanently_disabled_veteran = person("is_permanently_disabled_veteran", period)
        qualified_disabled_veteran_present = tax_unti.any(disabled_service_connected_veteran | permanently_disabled_veteran)
```

Reply:
need to check whether this only applied to head and spouse or also dependents

Reply:
According to the order of items listed on Form 1040ME, especially from items 12 to 14, I think `permanently and totally disabled veterans' credit` was calculated after the `base_credit` was calculated. And this part is applying upon the previously calculated total base credits which had already taken the filling status and the number of dependents into account.

Reply:
Yes that is correct, but the question remains, if the TaxUnit includes a senior who is a disabled veteran, who is not the main head or spouse of the household (e.g. father of the head/spouse) is this person considered for the credit or is limited to the head and spouse themselves

Reply:
A taxpayer can claim his/her parent(s) as dependents when he/she files under married jointly or as HOH, but the number of dependents does not determine whether the tax-filling entity will receive the extra credit or not.
Given your example where a senior disabled vet who is neither under Single, HOH, Married Jointly nor Qualified Surviving Spouse status, then he/she must be claimed as a dependent whose disability and military status are not considered when determining the eligibility for the VA credits on item 13, Form 1040ME. So for this filling entity, the extra VA credit won't be applied. However, if the senior disabled vet is filling on his/her own behalf and this would make him/her file under Single, HOH, Qualified Surviving Spouse or Married Jointly thus making the VA credit available.